platform/android/detect.py

```diff
@@ -280,8 +280,8 @@ def mySpawn(sh, escape, cmd, args, env):
                         '/toolchains/' + target_subpath + '/prebuilt/' + host_subpath + '/' + abi_subpath + '/lib'])
 
     env.Prepend(CPPPATH=['#platform/android'])
-    env.Append(CPPDEFINES=['ANDROID_ENABLED', 'UNIX_ENABLED', 'NO_FCNTL'])
-    env.Append(LIBS=['OpenSLES', 'EGL', 'GLESv3', 'GLESv2', 'android', 'log', 'z', 'dl'])
+    env.Append(CPPDEFINES=['ANDROID_ENABLED', 'UNIX_ENABLED', 'VULKAN_ENABLED', 'NO_FCNTL'])
+    env.Append(LIBS=['OpenSLES', 'EGL', 'GLESv2', 'vulkan', 'android', 'log', 'z', 'dl'])
```

Comment:
I guess Vulkan support should be enabled conditionally if `get_platform(env["ndk_platform"]) >= 24`. Unless we decide otherwise, it should remain possible for now to compile custom Android templates with only OpenGL ES.

API 18 to 23 are currently still close to 40% of the market share: https://developer.android.com/about/dashboards

Reply:
Note that if more work is needed to preserve the possibility to build against API 18-23, it can be left for a later PR.

Reply:
That's what I had in mind. I've updated the PR description to reflect this in the todo list.